packages/plugin_platform_interface/lib/plugin_platform_interface.dart

```diff
@@ -44,9 +44,19 @@ abstract class PlatformInterface {
   /// derived classes.
   ///
   /// @param token The same, non-`const` `Object` that will be passed to `verify`.
-  PlatformInterface({required Object token}) : _instanceToken = token;
+  PlatformInterface({required Object token}) {
+    _instanceTokens[this] = token;
+  }
 
-  final Object? _instanceToken;
+  /// Expando mapping instances of PlatformInterface to their associated tokens.
+  /// The reason we don't simply use a private field of type `Object?` is
```

Comment:
Nit: could you adjust this comment (here and below) to remove the use of "we"? I can't actually tell who "we" is in this context (see go/avoidwe).

Reply:
Done.